keyleak/site_scanner.py

```
@@ -29,165 +39,309 @@
+    crt = _crt_sh_subdomains(domain)
+    if crt:
+        candidates.update(crt)
+        _emit(on_progress, "subdomains", f"crt.sh returned {len(crt)} candidate names")
+
+    for sub in COMMON_SUBDOMAINS:
+        candidates.add(f"{sub}.{domain}")
+
+    # Keep only names that actually resolve, capped.
+    resolved: List[str] = []
+    # Always try the bare domain first so it leads the list.
+    ordered = [domain] + sorted(c for c in candidates if c != domain)
```

Ruff (0.15.14)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)